tests/python_tests/test_generate_api.py

```diff
@@ -361,25 +361,47 @@ def test_callback_batch_fail(callback):
         pipe.generate(['1', '2'], ov_genai.GenerationConfig(), callback)
 
 
-@pytest.mark.parametrize("callback", [print, user_defined_callback, lambda subword: print(subword)])
+class StremerWithResults:
```

Comment:
I think we need to add this to 
https://github.com/openvinotoolkit/openvino.genai/blob/b62145bdf0ca3c02f81a72f048ac8234210e7122/tests/python_tests/common.py#L328-L359

because it covers significantly more cases (but limit it to cases when we have a single batch and num_return_sequences is 1)

Reply:
I agree. But it's present only in master. This PR is for release branch. 
I will do that when cherry-pick to master.